README.md

```diff
@@ -35,6 +35,11 @@ sudo apt-get install git
 sudo apt-get install openjdk-7-jdk
 sudo apt-get install npm
 sudo apt-get install libfontconfig
+sudo npm -g install grunt-cli
```

Comment:
Thank you for sharing the experience!

What I was trying to say is  `mvn package` is designed to build zeppelin on clean environment, including web application, without requirement to have node.js&npm pre-installed there.

If it does not work like that on some environments - it has to be a bug well worth firing a JIRA issue.
 Having those things installed sounds like a reasonable workaround to me in such cases.